tests/juniper/juniper_base_protocol_test.py

```diff
@@ -1669,6 +1669,21 @@ def test_that_an_interface_cannot_be_deleted_while_there_is_still_an_rstp_config
 
         self.cleanup(reset_interface("ge-0/0/3"))
 
+    def test_that_force_up_can_be_configured(self):
+        self.edit({
+            "interfaces": [
+                {"interface": [
+                    {"name": "ge-0/0/1"},
+                    {"ether-options": {
+                        "ieee-802.3ad": {"lacp": {"force-up": ""}},
+                    }}]},
+            ]})
+        self.nc.commit()
+        get_interface_reply = self.nc.get_config(source="running", filter=dict_2_etree({"filter": {
+            "configuration": {"interfaces": {"interface": {"name": "ge-0/0/1"}}}}}))
+        assert_that(get_interface_reply.xpath("data/configuration"), has_length(1))
```

Comment:
something like that would asses more what you're looking for. 

        assert_that(get_interface_reply.xpath("data/configuration/interfaces/interface/ether-options/*")[0].tag, is_('ieee-802.3ad'))
        assert_that(get_interface_reply.xpath("data/configuration/interfaces/interface/ether-options/ieee-802.3ad/*")[0].tag, is_('lacp'))
        assert_that(get_interface_reply.xpath("data/configuration/interfaces/interface/ether-options/ieee-802.3ad/lacp/*")[0].tag, is_('force-up'))